packages/color-scale-block/src/dragAndDrop/DropZone.tsx

```diff
@@ -0,0 +1,80 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useDrop } from 'react-dnd';
+import { DraggableItem, DropZonePosition, OrderableListItem } from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+
+import { DropZoneProps } from '../types';
+
+export const DropZone = <T extends object>({
```

Comment:
We can be more precise than `object` (usage of `object` is a bad pratice and is a kind of equivalent of `any`)